pages/advanced-algorithms/available-algorithms/refactor.mdx

```diff
@@ -510,6 +510,192 @@ CALL refactor.invert(l) YIELD input, output RETURN input, output;
+                          - "outgoing" - the relationship with the bigger index in the path will be used for reconnection.
+                          - "merge" - the relationships will be merged, relationship type will be changed to "type2_type1" where type1 is the type of the relationship with the smaller index in the path. If both relationships have the same property the property in the reconnected relationship will be decided by `properties` in the config.
+                    - properties: string = "combine"
+                          - "discard" - property of the relationship with the smaller index in the path wins.
+                          - "overwrite" / "override" - property of the relationship with the bigger index in the path wins.
+                          - "combine" - merges properties by creating a list in which both property values are contained.
+{<h4> Output: </h4>}
+
+- `nodes: List[Node]` ➡ Nodes which were not deleted from the graph.
+- `relationships: List[Relationship]` ➡ Relationships which were not deleted from the graph.
+
+{<h4> Usage: </h4>}
+
+All the examples will be run on the following graph:
+
+```cypher
+CREATE (n1:Node {name: "Node 1"})-[:Rel1 {name: "Rel1"}]->(n2:Node {name: "Node 2"})-[:Rel2 {name: "Rel2"}]->(n3:Node {name: "Node 3"})-[:Rel3 {name: "Rel3"}]->(n4:Node {name: "Node 4"})-[:Rel4 {name: "Rel4"}]->(n5:Node {name: "Node 5"});
+```
+
+Query (incoming):
+```cypher
+MATCH p=(a:Node)-->(b:Node)-->(c:Node)-->(d:Node)-->(e:Node)
+WITH p, [b,d] as l CALL refactor.delete_and_reconnect(p, l, {relationshipSelectionStrategy: 'incoming'}) YIELD nodes, relationships RETURN nodes, relationships;
+```
+
+Result:
+```plaintext
+nodes: [{"id": 5, "labels": ["Node"], "properties": {"name": "Node 1"}, "type": "node"}, {"id": 7, "labels": ["Node"], "properties": {"name": "Node 3"}, "type": "node"}, {"id": 9, "labels": ["Node"], "properties": {"name": "Node 5"}, "type": "node"}]
+relationships: [{"id": 4, "start": 5, "end": 7, "label": "Rel1", "properties": {"name": "Rel1"}, "type": "relationship"}, {"id": 6, "start": 7, "end": 9, "label": "Rel3", "properties": {"name": "Rel3"}, "type": "relationship"}]
+```
+
+Query (outgoing):
+```cypher
+MATCH p=(a:Node)-->(b:Node)-->(c:Node)-->(d:Node)-->(e:Node)
+WITH p, [b,d] as l CALL refactor.delete_and_reconnect(p, l, {relationshipSelectionStrategy: 'outgoing'}) YIELD nodes, relationships RETURN nodes, relationships;
+```
+
+Result:
+```plaintext
+nodes: [{"id": 15, "labels": ["Node"], "properties": {"name": "Node 1"}, "type": "node"}, {"id": 17, "labels": ["Node"], "properties": {"name": "Node 3"}, "type": "node"}, {"id": 19, "labels": ["Node"], "properties": {"name": "Node 5"}, "type": "node"}]
+relationships: [{"id": 13, "start": 15, "end": 17, "label": "Rel2", "properties": {"name": "Rel2"}, "type": "relationship"}, {"id": 15, "start": 17, "end": 19, "label": "Rel4", "properties": {"name": "Rel4"}, "type": "relationship"}]
+```
+
+Query (merge + discard):
```

Comment:
Can you please help me... Can you write what exactly will happen in this version of the query... so not just the merge+ discard, but do you see how i wrote what will actually happen? so C/P and merging the config properties into a sentence